server/src/main/scala/scaladex/server/service/MoveReleasesSynchronizer.scala

```diff
@@ -0,0 +1,29 @@
+package scaladex.server.service
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+import ch.epfl.scala.index.model.misc.GithubStatus
+import ch.epfl.scala.index.newModel.NewProject
+import ch.epfl.scala.services.SchedulerDatabase
+import ch.epfl.scala.utils.ScalaExtensions.TraversableOnceFutureExtension
+
+class MoveReleasesSynchronizer(db: SchedulerDatabase)(implicit ec: ExecutionContext)
+    extends Scheduler("move-releases-synchronizer", 5.minutes) {
+  override def run(): Future[Unit] =
+    for {
+      allProjects <- db.getAllProjects()
```

Comment:
This methods gets all the project with their gthubInfo which can be huge because it contains all readme and issues...

It would be better to have a method `getAllProjectsStatus` that return `Seq[(Project.Reference, GithubStatus)]`

Or event better, after we split the columns of github_status:
```
getAllMovedProjects: Seq[Project.Reference]
```

Reply:
yes. will implement it when I change de coder for githubStatus